<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TerminalHelpMainDialog</class>
 <widget class="QDialog" name="TerminalHelpMainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>TerminalHelpMainDialog</string>
  </property>
  <widget class="QPushButton" name="pushButtonReturn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>360</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>

返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonEmergency">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>360</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>

求助</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCallCenter">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>360</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>

呼中心</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonHelp">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>360</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>

帮助</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>461</width>
     <height>301</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGraphicsView" name="graphicsViewIcon">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>231</width>
      <height>231</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/terminal_help/images/qt.png);</string>
    </property>
   </widget>
  </widget>
  <zorder>frame</zorder>
  <zorder>pushButtonReturn</zorder>
  <zorder>pushButtonEmergency</zorder>
  <zorder>pushButtonCallCenter</zorder>
  <zorder>pushButtonHelp</zorder>
  <zorder>listView</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
